### DISCUSSION TITLE
blog/validation-with-mediatr-pipeline-behavior-and-fluentvalidation

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#43] [General]
# blog/validation-with-mediatr-pipeline-behavior-and-fluentvalidation

Learn how to use MediatR Pipeline Behavior to implement Validations with FluentValidation in ASP.NET Core applications. This pattern is extremely powerful when clubbed with IExceptionHandler, and can help you build super clean solutions with an elegant way to handle Validations!

https://codewithmukesh.com/blog/validation-with-mediatr-pipeline-behavior-and-fluentvalidation?utm_source=linkedin

**Comment by shrinivas-chakravarthy** [comment]
Combine event sourcing to this combination of CQRS, MediatR Pipeline Behavior, FluentValidation, and IExceptionHandler. 

Also add Logging, Factory, Repository, UOW. This becomes a perfect boilerplate code template for new projects in .Net 8.

**Comment by shrinivas-chakravarthy** [comment]
Also add JWT authentication to the template
